imports/plugins/included/product-variant/client/templates/products/productGrid/productGrid.js

```diff
@@ -48,6 +49,14 @@ Template.productGrid.onRendered(function () {
       }
     });
   }
+  // Start Tour for New Users Automatically
+  const currentUser = Accounts.findOne(Meteor.userId());
+  const myIntro = introJs().setOption("showProgress", true)
+  .setOption("showStepNumbers", false);
+  if (Meteor.user().emails.length > 0 && !currentUser.takenTour) {
```

Comment:
I added it to check if the user has registered and this is being stored in an array